package/boot/uboot-envtools/files/mediatek_filogic

```diff
@@ -66,6 +66,7 @@ tplink,tl-xdr4288|\
 tplink,tl-xdr6086|\
 tplink,tl-xdr6088|\
 xiaomi,mi-router-wr30u-ubootmod|\
+xiaomi,mi-router-ax3000t-ubootmod|\
```

Comment:
Please keep entries sorter alphabetically (here and in other places you add them)

Reply:
Done